src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-position: center;
+    background-repeat: no-repeat;
```

Comment:
You can make it shorter. Use [background](https://developer.mozilla.org/en-US/docs/Web/CSS/background)